test/periphery/PixCashierUpgradeable.test.ts

```diff
@@ -95,21 +103,27 @@ describe("Contract 'PixCashierUpgradeable'", async () => {
     it("Is reverted if the contract is paused", async () => {
       await proveTx(pixCashier.setPauser(deployer.address));
       await proveTx(pixCashier.pause());
-      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount))
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_CONTRACT_IS_PAUSED);
     });
 
     it("Is reverted if the user has not enough tokens", async () => {
       await proveTx(brlcMock.mint(cashierClient.address, tokenAmount - 1));
-      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount))
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_TOKEN_TRANSFER_AMOUNT_EXCEEDS_BALANCE);
     });
 
+    it("Is reverted if the transaction id is not provided", async () => {
+      await proveTx(brlcMock.mint(cashierClient.address, tokenAmount));
+      await expect(pixCashier.connect(cashierClient).cashOut(tokenAmount, ethers.constants.HashZero))
+        .to.be.revertedWith(REVERT_MESSAGE_IF_TRANSACTION_ID_IS_NOT_PROVIDED);
+    });
+
     it("Transfers correct amount of tokens and changes cash-out balances accordingly", async () => {
       await proveTx(brlcMock.mint(cashierClient.address, tokenAmount));
       const oldCashOutBalance: number = await pixCashier.cashOutBalanceOf(cashierClient.address);
       await expect(async () => {
-        await proveTx(pixCashier.connect(cashierClient).cashOut(tokenAmount));
+        await proveTx(pixCashier.connect(cashierClient).cashOut(tokenAmount, TRANSACTION_ID));
```

Comment:
Codacy has a fix for the issue: [Replace `pixCashier.connect(cashierClient).cashOut(tokenAmount,·TRANSACTION_ID)` with `⏎··········pixCashier.connect(cashierClient).cashOut(tokenAmount,·TRANSACTION_ID)⏎········`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9795032)
```suggestion
        await proveTx(
```